src/calculator.js

```diff
@@ -1,47 +1,32 @@
-exports._check = () => {
+exports._check = (...args) => {
   // DRY up the codebase with this function
   // First, move the duplicate error checking code here
   // Then, invoke this function inside each of the others
   // HINT: you can invoke this function with exports._check()
+  args.forEach(arg => {
+    if (typeof arg !== 'number') {
+      throw new TypeError(`${arg} is not a number`)
```

Comment:
Missing semicolon.

###### See http://eslint.org/docs/rules/semi for details about this rule